.gitignore

```diff
@@ -7,7 +7,9 @@ tmp/
 .sass-cache/
 *.swp
 *.swo
+*.save
 public/uploads/*
+public/Chang/*
```

Comment:
You might want to ignore your personal files in `.git/info/exclude`.

Reply:
We should probably open a bug for this issue -- the app shouldn't be automatically creating personal directories during the import process, right?

Reply:
yeah, would be good to open an issue I apparently took @eitoball 's approach w/o even thinking about it

```
❯ cat ~/safecast/safecastapi/.git/info/exclude
# git ls-files --others --exclude-from=.git/info/exclude
# Lines that start with '#' are comments.
# For a project mostly in C, the following would be a good set of
# exclude patterns (uncomment them if you want to use them):
# *.[oa]
# *~

.envrc
public/mat/
```